samples/Mvc.RenderViewToString/EmailGraphHelper.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Mvc.RenderViewToString
+{
+    public class EmailReportHelper
```

Comment:
Meta-question: what is this class intending to demonstrate? What's it a surrogate for in a 'real' application?